docu/Concepts/TechnicalRequirements/UC_Introduction_of_Gradido-ID.md

```diff
@@ -0,0 +1,141 @@
+# Introduction of Gradido-ID
+
+## Motivation
+
+To introduce the Gradido-ID base on the requirement to identify an user account per technical key instead of using an email-address. Such a technical key ensures an exact identification of an user account without giving detailed information for possible missusage.
+
+Additionally the Gradido-ID allows to administrade any user account data like changing the email address or define several email addresses without any side effects on the identification of the user account.
+
+## Definition
+
+The formalized definition of the Gradido-ID can be found in the document [BenutzerVerwaltung#Gradido-ID](../BusinessRequirements/BenutzerVerwaltung#Gradido-ID).
+
+## Steps of Introduction
+
+To Introduce the Gradido-ID there are several steps necessary. The first step is to define a proper database schema with additional columns and tables followed by data migration steps to add or initialize the new columns and tables by keeping valid data at all.
+
+The second step is to decribe all concerning business logic processes, which have to be adapted by introducing the Gradido-ID.
+
+### Database-Schema
+
+#### Users-Table
+
+The entity users has to be changed by adding the following columns.
+
+| Column                   | Type   | Description                                                                            |
+| ------------------------ | ------ | -------------------------------------------------------------------------------------- |
+| gradidoID                | String | technical unique key of the user as UUID (version 4)                                   |
```

Comment:
It might be wise to market the alias as `GradidoID` and not the technical key - it has no inherited value and is not human readable

Reply:
I would prefer to use the wording:
GradidoID: UUID = never changeable Identifier
UserID:  alias = unique String = changeable Identifier